.github/ISSUE_TEMPLATE/feedback.md

```diff
@@ -0,0 +1,78 @@
+title: "[FEEDBACK]"
+labels: ["feedback"]
+assignees: [shawnalpay, jpradocueva]
+
+---
+
+### **Template Usage Notes**:
+1. All fields marked as **mandatory** [*] must be filled before submission.
+2. For **Supporting Data/Documentation**, ensure that linked files are accessible to relevant stakeholders.
+
+## 1. **Problem statement**
+Describe the problem or opportunity that this issue addresses. Explain the context and why it needs resolution.
+
+**Summary:**  
+_Briefly describe the problem or opportunity that this issue addresses and why it needs resolution._  
+> **Placeholder:** Briefly describe the problem or opportunity that needs resolution.
+
+**Which area does this issue relate to?**  
+_Select one of the provided options:_  
+- A: Missing FinOps feature, use case, or scenario
+- B: Existing FOCUS specification content
+- C: Other
+
+**Detailed Description:**  
+_Provide a detailed description of the problem or opportunity. Add context or any other relevant information that may help in addressing this issue._  
+> **Placeholder:** Provide a comprehensive description of the problem or opportunity, including context and any relevant details.
+
+## 2. **Use Cases**
+Please describe the use cases, whether FinOps-related or otherwise, that cannot be performed with the existing specification unless this issue is addressed.
+
+**What use cases, FinOps or others, can't be performed with the existing specification unless this issue is addressed?**  
+_Provide detailed descriptions of the use cases that can't be performed unless this issue is addressed._  
+> **Placeholder:** List the specific use cases that can't be performed due to the existing limitations.
+
+## 3. **FinOps Personas**
+List the FinOps personas this issue relates to.
+
+**Which FinOps personas does this issue relate to?**  
+_Indicate which FinOps personas this issue relates to._  
+> **Placeholder:** e.g., Practitioner, Executive, Finance, Engineering, Procurement, Operations, etc.
+
+## 4. **Providers**
+List provider groups or specific providers this issue relates to.
+
+**Which provider groups or specific providers does this issue relate to?**  
+_List provider groups (e.g., CSPs, SaaS) separated by semicolons (`;`), and specific providers within those groups separated by commas (`,`). If the issue applies to all providers within a group, you can simply specify the group name (e.g., 'All CSPs')._  
+> **Placeholder:** e.g., All CSPs; SaaS: Snowflake, Salesforce
+
+## 5. **Criticality Scale**
+Indicate how critical this issue or feature request is for your organization using the scale provided below.
+
+**On a scale of 1 - 4, how critical is this for your organization?**  
+_Select one of the provided options:_  
+- 1: Blocks my organization from adopting FOCUS
+- 2: Important for adoption in the next 3-6 months
+- 3: Important for adoption in the next 6-12 months
+- 4: Suggestion, not planning to use FOCUS soon
```

Comment:
I'm not sure if the Criticality is meant to indicate 
- the urgency of resolving the issue 
- or the organization’s own FOCUS adoption plans. 

Assuming we're focusing on the urgency of resolution, I propose the following suggestion:

```suggestion
- 1: Critical - Blocks my organization from adopting FOCUS
- 2: Important - Ideally resolved within the next 3-6 months
- 3: Important - Ideally resolved within the next 6-12 months
- 4: Suggestion - Resolution not urgent
```

Reply:
Agree with suggested change